### PR TITLE
Thanks

### PR DESCRIPTION
**Pull request by vladimirfomene**
"Please check this out @casey-collab".

### CODE REVIEW

File: english/sing-song/vlad.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[Thank God. Am so happy to be at the end.] (www.crazyvlad.com)
```
Comment (casey-collab) [inline english/sing-song/vlad.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/sing-song/vlad.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/sing-song/vlad.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/sing-song/vlad.md (hunk 1 above)
Comment (casey-collab) [inline english/sing-song/vlad.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/sing-song/vlad.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/thankYou/vladStory.md [hunk 2]
```diff
@@ -0,0 +1 @@
+[Thank God. Am so happy to be at the end.] (https://github.com/vladimirfomene/)
```
Comment (casey-collab) [inline english/thankYou/vladStory.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/sing-song/vlad.md (hunk 1 above)
Comment (casey-collab) [inline english/sing-song/vlad.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
It looks like you wish https://github.com/udacity/create-your-own-adventure/pull/892 to be merged instead of this pull request, so I'm closing this one. If I've misunderstood, please let me know.

**Pull request closed without merging**